src/asm2wasm.h

```diff
@@ -150,6 +150,13 @@ struct AstStackHelper {
 
 std::vector<Ref> AstStackHelper::astStack;
 
+static bool startsWith(const char* string, const char *prefix) {
+  while (1) {
+    if (*prefix == 0) return true;
+    if (*string++ != *prefix++) return false;
```

Comment:
Should also probably return false if `*string == 0` for completeness, e.g. `startsWith("foo", "foobar")`